src/components/ModelActions/ModelActions.tsx

```diff
@@ -0,0 +1,54 @@
+import { ContextualMenu } from "@canonical/react-components";
+import { Link } from "react-router";
+
+import useCanConfigureModel from "hooks/useCanConfigureModel";
+import { useQueryParams } from "hooks/useQueryParams";
+import { getIsJuju } from "store/general/selectors";
+import { useAppSelector } from "store/store";
+import { rebacURLS } from "urls";
+
+import { Label, type Props } from "./types";
+
+const ModelActions = ({ modelName, activeUser }: Props) => {
+  const [, setPanelQs] = useQueryParams<{
+    model: string | null;
+    panel: string | null;
+  }>({
+    model: null,
+    panel: null,
+  });
+  const canConfigureModel = useCanConfigureModel(false, modelName, activeUser);
+  const isJuju = useAppSelector(getIsJuju);
+
+  return (
+    <ContextualMenu
```

Comment:
I know that the designs aren't final, but we can can make the button appear like the designs if we use the `p-icon--contextual-menu` icon and set the button appearance to base and also right align the action column.

Also, we should remove the margin at the bottom of the toggle button as it's adding extra space at the bottom of each row.